.prettierrc

```diff
@@ -1,6 +0,0 @@
-{
-  "endOfLine": "auto",
```

Comment:
Looks like removing this is now failing Windows CI. Maybe prettier is merging these configs?

Reply:
No, that file has priority. I will re-add it and remove the other one.